docs/trace_service_features.md

```diff
@@ -0,0 +1,114 @@
+# Trace Feature Coverage by Service
+
+## TOC
+
+- [Ad Service](#ad-service)
+- [Cart Service](#cart-service)
+- [Checkout Service](#checkout-service)
+- [Currency Service](#currency-service)
+- [Email Service](#email-service)
+- [Feature Flag Service](#feature-flag-service)
+- [Frontend Service](#frontend-service)
+- [Payment Service](#payment-service)
+- [Product Catalog Service](#product-catalog-service)
+- [Recommendation Service](#recommendation-service)
+- [Shipping Service](#shipping-service)
+
+## OpenTelemetry Feature Set by Services
+
+### Emoji Legend
+
+- Completed: :100:
+- Not Present (Yet): :construction:
+
+### [Ad Service](../src/adservice/README.md)
+
+tba
+
+### [Cart Service](../src/cartservice/README.md)
+
+tba
+
+### [Checkout Service](../src/checkoutservice/README.md)
+
+tba
+
+### [Currency Service](../src/currencyservice/README.md)
+
+tba
+
+### [Email Service](../src/emailservice/README.md)
+
+- :100: [Instrumentation
+  Libraries](https://opentelemetry.io/docs/concepts/instrumenting-library/)
+- :100: [Manual Span
+  Creation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#manual-instrumentation)
+- :100: [Span Data
+  Enrichment](https://opentelemetry.io/docs/instrumentation/net/manual/#add-tags-to-an-activity)
+- :100: Interprocess Context Propagation
+- :construction: [Intra-service Context
+  Propagation](https://opentelemetry.io/docs/instrumentation/java/manual/#context-propagation)
```

Comment:
why link to a java specific doc?

Reply:
I'd link to the spec!
or the spec via docs page : https://opentelemetry.io/docs/reference/specification/context/api-propagators/